attributes/default.rb

```diff
@@ -174,4 +174,8 @@
 default['kube-hops']['coredns-autoscaler']['target-deployment']        = "Deployment/coredns"
 
 default['kube-hops']['nvidia-device-plugin']['version']                =  "0.13.0"
-default['kube-hops']['nvidia-device-plugin']['url']                    = node['download_url'] + "/kube/nvidia/device-plugin/#{node['kube-hops']['nvidia-device-plugin']['version']}/nvidia-device-plugin.yml"
\ No newline at end of file
+default['kube-hops']['nvidia-device-plugin']['url']                    = node['download_url'] + "/kube/nvidia/device-plugin/#{node['kube-hops']['nvidia-device-plugin']['version']}/nvidia-device-plugin.yml"
+
+default['kube-hops']['device-manager']['fuse-max-devices']             = 30
+default['kube-hops']['fuse']['assets_dir']                             = node['kube-hops']['hops-system']['base_dir'] + "/fuse"
```

Comment:
It feels like `assets_dir` could be reused in the future for something else, so I would suggest making the attribute `['kube-hops']['assets_dir']['fuse']`